pkgs/development/python-modules/loguru-logging-intercept/default.nix

```
@@ -0,0 +1,30 @@
+{
+  lib,
+  buildPythonPackage,
+  setuptools,
+  loguru,
+  fetchFromGitHub,
+}:
+buildPythonPackage {
+  pname = "loguru-logging-intercept";
+  version = "0.1.4-unstable";
+
+  src = fetchFromGitHub {
+    owner = "MatthewScholefield";
+    repo = "loguru-logging-intercept";
+    rev = "9252ac889d4af08fd11d760d92d56a9de93a004a";
+    hash = "sha256-lHACnXhXblNfzBYZ/3BTVupp6b9257jDN5kY3xvfVUg=";
+  };
+
+  build-system = [ setuptools ];
```

```suggestion
  pyproject = true;

  build-system = [ setuptools ];
```

Please add `pyproject = true` to avoid the legacy build.

> Note that the pyproject format falls back to using setuptools, so you can use pyproject = true even if the package only has a setup.py.

https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonpackage-parameters-buildpythonpackage-parameters